plugins/catalog/src/components/SystemDiagramCard/SystemDiagramCard.tsx

```diff
@@ -128,7 +128,9 @@ function RenderNode(props: DependencyGraphTypes.RenderNodeProps<any>) {
           alignmentBaseline="baseline"
           style={{ fontWeight: 'bold' }}
```

Comment:
Have you tried if it works to use css ellipsis handling instead of manual truncation?

Something like

```ts
maxWidth: '200px',
textOverflow: 'ellipsis',
overflow: 'hidden',
whiteSpace: 'nowrap',
```

Reply:
I had a go with some CSS solutions, as it would be nicer, I agree. But I don't think this can apply since the `text` is part of the SVG, so I couldn't get it to apply wrapping CSS properties with an effect. 

Source: https://css-tricks.com/svg-properties-and-css/